docs/tutorial/security.md

```diff
@@ -114,8 +114,9 @@ You should at least follow these steps to improve the security of your applicati
 15. [Do not use `shell.openExternal` with untrusted content](#15-do-not-use-shellopenexternal-with-untrusted-content)
 16. [Use a current version of Electron](#16-use-a-current-version-of-electron)
 17. [Validate the `sender` of all IPC messages](#17-validate-the-sender-of-all-ipc-messages)
-18. [Avoid usage of the `file://` protocol and prefer usage of custom protocols](#18-avoid-usage-of-the-file-protocol-and-prefer-usage-of-custom-protocols)
-19. [Check which fuses you can change](#19-check-which-fuses-you-can-change)
+18. [Do not expose Electron APIs to untrusted web content](#18-do-not-expose-electrons-apis-to-untrusted-web-content)
```

Comment:
Can we add this to the bottom of the list so that any existing hyperlinks stay correct for the existing 18 and 19 items?

Reply:
I've updated the existing links and wanted to optimize more for us not jumping back and forth between thematic areas - are you worried about links from outside our own repo? I could be convinced that maintaining old stuff trumps readability of new stuff.

Reply:
Yeah, it was more so that hyperlink consistency remains the same. I'm okay making that tradeoff, though. 👍

Reply:
In my application's source code comments, I sometimes link to anchor links from the security guide. These links will break through modifications like this one.

If we go with thematic areas (which I'm fine with), we should remove numbers from all hyperlinks in the guide completely. Then we'll only break links once and going forward they won't break again. Whereas with numbers, they might break again every 6 months.